docs/reference/announcements.md

```diff
@@ -81,6 +81,10 @@ If you do not update the cluster by August 30th 2024, we will update the cluster
 
 Camunda 8 Self-Managed clusters are not affected by this.
 
+### Support for Amazon OpenSearch for Optimize
+This release supports an extended feature set in Optimize that is available on OpenSearch. Full support is committed for the next release in Jan 2025.
```

Comment:
Is the referenced feature set in OpenSearch (and we now support it), or in Optimize (and it uses OpenSearch)? It's a little unclear. Do we have any additional pages we can link to for more information/to clarify?

Reply:
@conceptualshark, there are no additional pages I can think of. This announcement is similar to the one we used for previous releases. What are you looking exactly for? A list of features that are supported on Optimize?

Reply:
```suggestion
This release extends the OpenSearch features supported by Optimize. Full support is committed for the next release in January 2025.
```

^ Is this accurate? It would be a little clearer if so. No need for links if we don't have anything supporting.

Reply:
Yes - that sounds good.